revolv/project/models.py

```diff
@@ -426,6 +448,43 @@ def categories(self):
         return [category.title for category in self.category_set.all()]
 
     @property
+    def updates(self):
+        return self.update_set.all()
+
+    def add_update(self, text):
+        update = ProjectUpdate(update_text=text, project=self)
+        update.save()
+
+class ProjectUpdate(models.Model):
+    update_text = models.TextField(
```

Comment:
But like, why can't you call the field `text` instead of `update_text`? `update_text is pretty redundant`